src/DocumentFormat.OpenXml/GeneratedCode/schemas_openxmlformats_org_drawingml_2006_main.g.cs

```diff
@@ -12758,7 +12758,7 @@ public StringValue Name
         }
 
         /// <summary>
-        /// <para>id</para>
+        /// <para>id, this property is only available in Office 2013 and later.</para>
```

Comment:
File name indicates schema from year 2006. Is it correct that this parameter is "only available in Office 2013 and later"?

Reply:
Same issue here; thm15:id property added to the existing a:theme (2010 or earlier) root element. Hence updating in the 2006  namespace-named generated code.